pkg/kubelet/qos/util/qos.go

```diff
@@ -19,12 +19,7 @@ package util
 import (
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/resource"
-)
-
-const (
-	Guaranteed = "Guaranteed"
-	Burstable  = "Burstable"
-	BestEffort = "BestEffort"
+	qostypes "k8s.io/kubernetes/pkg/kubelet/qos/types"
```

Comment:
Yeah we will have import cycles because of the way the code is currently structured, but making the changes you suggest would make sense.